src/pybind/mgr/dashboard/frontend/src/styles/ceph-custom/_toast.scss

```
@@ -1,3 +1,29 @@
+@use './src/styles/defaults/bootstrap-defaults' as bd;
+
+::ng-deep #toast-container {
```

we can't use `::ng-deep` outside of components:
```suggestion
#toast-container {
```